mappings/net/minecraft/client/gui/screen/realms/RealmsPlayerScreen.mapping

```diff
@@ -16,12 +22,41 @@ CLASS net/minecraft/unmapped/C_owmmvzoi net/minecraft/client/gui/screen/realms/R
 	CLASS C_ebbscyhc InvitedObjectEntry
 		FIELD f_ezlfcvdo makeOperatorButton Lnet/minecraft/unmapped/C_buwziidm;
 		FIELD f_iwajvqpi removeOperatorButton Lnet/minecraft/unmapped/C_buwziidm;
+		FIELD f_larmhgrv REMOVE_PLAYER Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_rbylnstn REMOVE_OPERATOR Lnet/minecraft/unmapped/C_ncpywfca;
 		FIELD f_ssqjmwwi removePlayerButton Lnet/minecraft/unmapped/C_buwziidm;
+		FIELD f_ullrlikm INVITE_NORMAL Lnet/minecraft/unmapped/C_rdaqiwdt;
+		FIELD f_utplludm OPERATOR_SPRITE_WIDTH I
+		FIELD f_vjdprtow OPERATOR_SPRITE_HEIGHT I
+		FIELD f_wrjftsht INVITE_REMOVE Lnet/minecraft/unmapped/C_rdaqiwdt;
+		FIELD f_wzsxnmcl INVITE_OP Lnet/minecraft/unmapped/C_rdaqiwdt;
 		FIELD f_yrxdfmqv playerInfo Lnet/minecraft/unmapped/C_uckflzvp;
+		FIELD f_zbgaczmk MAKE_OPERATOR Lnet/minecraft/unmapped/C_ncpywfca;
 		METHOD <init> (Lnet/minecraft/unmapped/C_owmmvzoi;Lnet/minecraft/unmapped/C_uckflzvp;)V
 			ARG 2 playerInfo
+		METHOD m_emysazsp updateOps (Lnet/minecraft/unmapped/C_okicnizs;)V
+			ARG 1 ops
+		METHOD m_hahiqagf deopPlayer (I)V
+			ARG 1 index
+		METHOD m_hvkguzat (Lnet/minecraft/unmapped/C_uckflzvp;Ljava/util/function/Supplier;)Lnet/minecraft/unmapped/C_npqneive;
+			ARG 1 textSupplier
+		METHOD m_jagblmvl (Lnet/minecraft/unmapped/C_uckflzvp;Ljava/util/function/Supplier;)Lnet/minecraft/unmapped/C_npqneive;
+			ARG 1 textSupplier
 		METHOD m_lcrgzomf (ILnet/minecraft/unmapped/C_buwziidm;)V
 			ARG 2 button
+		METHOD m_lerxpmdr updateOperatorButtons ()V
+		METHOD m_ntvsespc getOperatorButton ()Lnet/minecraft/unmapped/C_buwziidm;
+		METHOD m_nycgvaso uninvitePlayer (I)V
```

Comment:
```suggestion
		METHOD m_nycgvaso removePlayer (I)V
```

Reply:
I chose `uninvitePlayer` because of the strings `"Couldn't uninvite user"` and `"mco.configure.world.uninvite.player"` and the delegation to `RealmsClient#uninvite`. I'm not sure if the code overlaps with the code to remove players that are already members. mojmap is `uninvite`

Reply:
Ah I see. I was looking at where it was called from the buttons, and that all has `remove` associated with it.